src/api/configuration.cpp

```diff
@@ -169,6 +169,17 @@ namespace mamba
                                 (then restart or source the contents of the shell init script))");
             }
 
+            if (!fs::exists(prefix))
+            {
+                fs::create_directories(prefix / "conda-meta");
+                fs::create_directories(prefix / "envs");
```

Comment:
might be better to use path::touch? https://github.com/mamba-org/mamba/blob/79d3342a04bc57ef21b99c0b926b8a46b24dbafa/include/mamba/core/fsutil.hpp#L48

Reply:
thanks for the pointer! It should be fixed now